components/brave_wallet/browser/zcash/zcash_grpc_utils.cc

```diff
@@ -18,35 +18,6 @@ constexpr size_t kGrpcHeaderSize = 5;
 constexpr char kNoCompression = 0;
 }  // namespace
 
-std::string GetPrefixedProtobuf(const std::string& serialized_proto) {
-  std::string result(kGrpcHeaderSize, 0);
-  result[0] = kNoCompression;
-  base::WriteBigEndian<uint32_t>(&result[1], serialized_proto.size());
-  result.append(serialized_proto);
-  return result;
-}
-
-// Resolves serialized protobuf from length-prefixed string
-std::optional<std::string> ResolveSerializedMessage(
-    const std::string& grpc_response_body) {
-  if (grpc_response_body.size() < kGrpcHeaderSize) {
-    return std::nullopt;
-  }
-  if (grpc_response_body[0] != kNoCompression) {
-    // Compression is not supported yet
-    return std::nullopt;
-  }
-  uint32_t size = 0;
-  base::ReadBigEndian(
-      reinterpret_cast<const uint8_t*>(&(grpc_response_body[1])), &size);
-
-  if (grpc_response_body.size() != size + kGrpcHeaderSize) {
-    return std::nullopt;
-  }
-
-  return grpc_response_body.substr(kGrpcHeaderSize);
-}
-
 GRrpcMessageStreamHandler::GRrpcMessageStreamHandler() = default;
 GRrpcMessageStreamHandler::~GRrpcMessageStreamHandler() = default;
 
```

Comment:
As far as I understand we also should move GRrpcMessageStreamHandler to zcash decoder

Reply:
GRrpcMessageStreamHandler is linked to SimpleUrlLoader, it just extracts chunks from incoming stream to decode. Not linked to decoding itself.